app/views/events/index.html.erb

```diff
@@ -37,7 +37,8 @@
             <td align="center"><%= visibility_switch(event, "EventRecord") %></td>
           <% end %>
           <td align="right" class="text-nowrap">
-            <%= link_to "Bearbeiten", edit_event_path(event.id), class: "btn btn-sm btn-secondary" %>
+            <% edit_label = editable_by_user?(event) ? "Bearbeiten" : "Ansehen" %>
```

Comment:
`link_to_label`, because it is not only `edit`?

Reply:
but we are rendering the edit form and linking to the edit controller...